internal/provider/resource_gitlab_project_milestone_test.go

```diff
@@ -0,0 +1,97 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccGitlabProjectMilestone_basic(t *testing.T) {
+
+	rInt1, rInt2 := acctest.RandInt(), acctest.RandInt()
+	project := testAccCreateProject(t)
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabProjectMilestoneDestroy,
+		Steps: []resource.TestStep{
+			{
+				// create Milestone with required values only
+				Config: fmt.Sprintf(`
+                resource "gitlab_project_milestone" "this" {
+                  project = "%v"
+                  title   = "test-%d"
+                }`, project.PathWithNamespace, rInt1),
```

Comment:
NIT: for better readability:

```suggestion
				Config: fmt.Sprintf(`
				resource "gitlab_project_milestone" "this" {
                                   project = "%v"
                                   title   = "test-%d"
                                 }`, project.PathWithNamespace, rInt1),
```

Reply:
Hm,  I thought `make reviewable` replace this. Okay...